src/kern/arm/bsp/drivers/mru.cpp

```diff
@@ -0,0 +1,246 @@
+INTERFACE [arm && pic_gic]:
+
+#include "gic.h"
+#include "gic_v3.h"
+#include "initcalls.h"
```

Comment:
The initcalls header is not needed by the MRU code AFAICT. It needs to be retained in `src/kern/arm/bsp/s32z/pic-arm-s32z.cpp`, though.